# time stamp for this status message
time stamp

# id of this request
uint32 id

# type of request. This is either 'input' or 'output'
string type

# the package type
PackageConfiguration pkg_config

# time when this request was created
time create_time

# message describing the current status of this request
string status

